app/views/patients/_patient_information.html.erb

```diff
@@ -113,4 +113,4 @@
       </div>
     </div>
   <% end %>
-</section>
+</section>
```

Comment:
Not a blocker here, but generally you want to have newlines at the end of files. This is the source of a lot of linux arguing/opinions, but it is kind of a good practice! Here's some of said arguing: https://unix.stackexchange.com/questions/23903/should-i-end-my-text-script-files-with-a-newline